dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/imports/TrackerImportControllerTest.java

```diff
@@ -370,7 +373,7 @@ void verifyShouldThrowWhenJobReportNotFound()
         mockMvc.perform( get( ENDPOINT + "/jobs/" + uid + "/report" )
             .content( "{}" )
             .contentType( MediaType.APPLICATION_JSON )
-            .accept( MediaType.APPLICATION_JSON ) ).andExpect( status().isNotFound() )
+            .accept( MediaType.APPLICATION_JSON ) )
```

Comment:
After adding the CrudControllerAdvice class, this status is not reflecting the WebMessage status anymore.

Reply:
Can't really follow you. If the report does not exist I think we want to keep having this result in a 404.

Reply:
As we discussed, these type of tests are not returning the correct status as the `mockMvc` should set the WebMessage Advice to enhance the response with the correct status